configs/katello/4.1.yaml

```diff
@@ -40,7 +40,31 @@
   :foreman_virt_who_configure:
     :branch: 0.5.6
     :repo: https://github.com/theforeman/foreman_virt_who_configure.git
+  :smart_proxy_container_gateway:
+    :branch: 1.0.4
+    :repo: https://github.com/Katello/smart_proxy_container_gateway.git
+  :katello-selinux:
+    :branch: 4.0.0
+    :repo: https://github.com/Katello/katello-selinux.git
 :ignores:
+  - 32155 # tracker
```

Comment:
Does it make sense to make cherry picking smart enough to always ignore tracker issues?

Reply:
Probably